web/public/locales/en.json

```diff
@@ -90,6 +90,24 @@
       "total-electricity-consumption": "Total electricity consumption by source"
     }
   },
+  "estimation-feedback": {
+    "title": {
+      "state-1": "Help us improve!",
+      "state-3": "Thank you!"
+    },
+    "subtitle": {
+      "state-1": "Please rate the following statement.",
+      "state-3": "Your response helps us understand and improve the quality of our communication and methodology."
+    },
+    "rate-question": "The description above was easy to understand:",
```

Comment:
Users may think, what description above? If it's only going to be feedback of the estimation card we could say "The description of the data estimation was easy to understand"

Reply:
Good point. I talked to Alex and we agree so I've changed it to be "The description of the data estimation was easy to understand"